### PR TITLE
Add Hab to the expeditor script

### PR DESCRIPTION
**Pull request by kagarmoe**
There's a solid chance that I made a wrong guess for the keyname.

Signed-off-by: kagarmoe <kgarmoe@chef.io>

### CODE REVIEW

File: .expeditor/update_hugo_modules.sh [hunk 1]
```diff
@@ -21,19 +21,22 @@ elif [ "${EXPEDITOR_PRODUCT_KEY}" == "inspec" ]; then
 elif [ "${EXPEDITOR_PRODUCT_KEY}" == "automate" ]; then
   subdirectory="components/docs-chef-io"
   org="chef"
+elif [ "${EXPEDITOR_PRODUCT_KEY}" == "habitat" ]; then
+  subdirectory="components/docs-chef-io"
```
Comment (IanMadd) [inline .expeditor/update_hugo_modules.sh:25]:
```suggestion
  subdirectory="components/habitat-chef-io"
```

**Comment by IanMadd** [comment]
We shouldn't merge this until we've got hab docs set up in docs.chef.io.

**Comment by kagarmoe** [comment]
But it has all the double-quotes that we need to pass shellcheck. Would adding it here cause us to go live before we want to?

**Comment by IanMadd** [comment]
Never mind. This doesn't update Expeditor's config.yaml to watch for Habitat updates. I was getting ahead of myself.

**Review by IanMadd — APPROVED** [review]
tenor-115394545

**Pull request merged**